docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -113,6 +113,23 @@
 
 - `spot` (Spot) - If set use a spot instance during build; spot configuration settings only apply to the virtual machine launched by Packer and will not be persisted on the resulting image artifact.
 
+Following is an example:
+
+  In JSON
```

Comment:
Just just a heads up this is a generated file (see line 1) that is generated based on the comments in `builder/azure/arm/config.go` https://github.com/hashicorp/packer-plugin-azure/blob/353eea159082d35fcbaebaafb6708b57aee2fa2a/builder/azure/arm/config.go#L279 here

Here is an example of an HCL/JSON example in this format https://github.com/hashicorp/packer-plugin-azure/blob/353eea159082d35fcbaebaafb6708b57aee2fa2a/builder/azure/arm/config.go#L173-L197

Than you run `make generate` which will generate the docs-partials.  If I run `make generate` right now on your branch, it deletes your additional changes

Reply:
Thanks for the review and clear explanation! I've added a commit with the source update (and the result of `make generate`).

Is that ok ?